contrib/uriparser/utils/utils_uriparser.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CONTRIB_URIPARSER_UTILS_UTILS_ZIP_H_
+#define CONTRIB_URIPARSER_UTILS_UTILS_ZIP_H_
+
+#include <vector>
+
+#include "contrib/uriparser/sandboxed.h"
+#include "sandboxed_api/util/status_macros.h"
+
+class Uriparser {
+ public:
+  explicit Uriparser(UriparserSandbox* sandbox, const std::string& uri)
+      : sandbox_(CHECK_NOTNULL(sandbox)), api_(sandbox_), c_uri_(uri.c_str()) {
+    status_ = ParseUri();
+  };
+
+  ~Uriparser();
+
+  absl::Status GetStatus();
+  absl::Status NormalizeSyntax();
+  absl::Status NormalizeSyntax(int norm_mask);
+
+  absl::StatusOr<std::string> GetURI();
```

Comment:
More naming: "prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC()."

This would also be more consisten with `ParseUri()`.

Reply:
Fixed.